src/axes/categoryAxis.ts

```diff
@@ -300,6 +300,12 @@ namespace Plottable.Axes {
       let xTranslate = this.orientation() === "right" ? this._tickSpaceRequired() : 0;
       let yTranslate = this.orientation() === "bottom" ? this._tickSpaceRequired() : 0;
       Utils.DOM.translate(this._tickLabelContainer, xTranslate, yTranslate);
+
+      // hide ticks and labels that overflow the axis
+      this._showAllTickMarks();
+      this._showAllTickLabels();
+      this._hideOverflowingTickLabels();
+      this._hideTickMarksWithoutLabel();
```

Comment:
This won't be an issue because those labels aren't hidden with the CSS visibility rule. Instead the SVGTypewriter just doesn't draw the text because there isn't enough space. But the label is still technically "visible".